app/models/manageiq/providers/openstack/cloud_manager/refresh_parser.rb

```diff
@@ -66,9 +66,12 @@ def ems_inv_to_hashes
       # get_hosts
       get_images
       get_servers
+      # TODO: change spec to remove this
+      #
       get_volumes
```

Comment:
Yes, will remove these eventually. The reason to keep it is to avoid Travis CI build failure.